homeassistant/components/ihc/util.py

```diff
@@ -0,0 +1,10 @@
+"""Useful functions for the IHC component."""
+
+import time
+
+
+def pulse(ihc_controller, ihc_id: int):
+    """Send a short on/off pulse to an IHC controller resource."""
+    ihc_controller.set_runtime_value_bool(ihc_id, True)
+    time.sleep(0.1)
```

Comment:
We don't allow sync sleep in our thread pool. If sleep is necessary, it needs to be done with asyncio.sleep in a coroutine.

Reply:
I hope I fixed this the right way.